### PR TITLE
Updated Static Route Minigraph Attribute property

### PR DESCRIPTION
**Pull request by abdosi**
What I did:
Updated Static Route Attribute in Minigraph. NGS Minigraph has define semantics of static route differently.
See below for differences:-

Microsoft ADO: https://msazure.visualstudio.com/One/_workitems/edit/17956325

Before

```
<AssociatedTo>8.0.0.1/32</AssociatedTo>
<Address>192.168.1.2,192.168.2.2</Address>
<AttachTo>PortChannel40,PortChannel50</AttachTo>
```

Now:
```

<Address>8.0.0.1</Address>
<AttachTo>PortChannel40,192.168.1.2;PortChannel50,192.168.2.2</AttachTo>
```

Why I did:

Needed for Chassis use case.

How I verify:
Updated UT
Manual Verification

### CODE REVIEW

**Comment by abdosi** [comment]
@arlakshm / @prsunny can you help review this

**Comment by prsunny** [comment]
hi @abdosi , could you please provide more details on the changes in description?

**Comment by abdosi** [comment]
> hi @abdosi , could you please provide more details on the changes in description?

@prsunny updated with more details

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -570,11 +570,22 @@ def parse_dpg(dpg, hname):
...
                 elif ipnh.find(str(QName(ns, "Type"))).text == 'StaticRoute':
-                    prefix = ipnh.find(str(QName(ns, "AssociatedTo"))).text
-                    ifname = ipnh.find(str(QName(ns, "AttachTo"))).text
-                    nexthop = ipnh.find(str(QName(ns, "Address"))).text
-                    advertise = ipnh.find(str(QName(ns, "Advertise"))).text
-                    static_routes[prefix] = {'nexthop': nexthop, 'ifname': ifname, 'advertise': advertise}
+                    prefix = ipnh.find(str(QName(ns, "Address"))).text
+                    ifname = []
+                    nexthop = []
+                    for nexthop_tuple in ipnh.find(str(QName(ns, "AttachTo"))).text.split(";"):
```
Comment (prsunny) [inline src/sonic-config-engine/minigraph.py:576]: Do we need to handle backward compatibility? Like, handle a case if old minigraph is loaded?

Comment (abdosi) [reply src/sonic-config-engine/minigraph.py:576]: no previous format was defined by us for chassis. Once this PR is merged we will update sonic-mgmt template minigraph_dpg.j2 to use this updated format.  So `deploy-mg` should start using new format.

**Review by prsunny — APPROVED** [review]

**Review by arlakshm — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Pull request merged**